### PR TITLE
Helm chart.yaml fix PR

### PR DESCRIPTION
**Pull request by jonahjon**
Signed-off-by: jonahjon <jonahjones094@gmail.com>

*Issue #, if available:*

*Description of changes:*

*Testing (if applicable):*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: release/go.mod [hunk 1]
```diff
@@ -8,67 +8,160 @@ require (
 	github.com/aws/eks-distro-build-tooling/release v0.0.0-20211103003257-a7e2379eae5e
 	github.com/fsouza/go-dockerclient v1.7.2
 	github.com/ghodss/yaml v1.0.0
+	github.com/go-logr/logr v1.2.0
```
Comment (abhinavmpandey08) [inline release/go.mod:11]: Can you try running `go mod tidy` in the `release` directory to make sure all these dependencies are actually needed?

Comment (pokearu) [reply release/go.mod:11]: Also looks like some of the dependencies versions have been bumped up. Are we sure we need that?

Comment (jonahjon) [reply release/go.mod:11]: I was using logr for the helm driver, since it's the same pattern we use for package controller in there.

Comment (panktishah26) [reply release/go.mod:11]: Can you try running `go mod tidy` in the `release` directory? It has some go.mod file changes. Other than this everything else looks good.

**Review by abhay-krishna — APPROVED** [review]
/lgtm
/approve

**Pull request merged**